cves/CVE-2019-11338.yml

```diff
@@ -186,8 +195,10 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: |
+    This vulnerability can be exploited by using duplicate first slices in a multimedia file, which could
```

Comment:
Ah - not sure what a "first slice" is, but the opening description could use this detail and then some explanation of what that is.